config/config_test.go

```diff
@@ -112,6 +112,8 @@ func Test_NewConfig(t *testing.T) {
 				EtcdTimeout:                             2 * time.Second,
 				AppendFilters:                           &defaultFiltersFlags{},
 				PrependFilters:                          &defaultFiltersFlags{},
+				CloneRoute:                         &routeChangerConfig{},
```

Comment:
is there some formatting problem, or just github displays it a bit off for me?

Reply:
fixed

Reply:
this was fixed